documentation/docs/14-configuration.md

```diff
@@ -232,6 +232,7 @@ Enables or disables the client-side [router](#page-options-router) app-wide.
 
 An object containing zero or more of the following values:
 
+- `register` - set to `false` to disable automatic service worker registration
```

Comment:
```suggestion
- `register` - whether to enable automatic service worker registration. Default is `true`
```

Reply:
I'm following a similar wording style to the `prerender.enabled` config, and we don't really mention the default value for the config anywhere since it's already displayed in the code block at the start, so I don't think the last part is necessary